batch_writer.go

```diff
@@ -34,6 +34,17 @@ func (w *BatchWriter) WriteRowBatch(batch *RowBatch) error {
 			return nil
 		}
 
+		lastRow := batch.Values()[len(batch.Values())-1]
+		startPkpos, err := batch.Values()[0].GetUint64(batch.PkIndex())
+		if err != nil {
+			return err
```

Comment:
I know this is just moved code, but we're not logging anything beyond the error, I guess we really don't expect this to fail?

Reply:
Yeah this really shouldn't fail as it was set by the dataiterator which should have the correct type.